packages/fury-adapter-oas3-parser/lib/parser/oas/parseOperationObject.js

```diff
@@ -33,16 +33,40 @@ function createTransactions(namespace, member, operation) {
   });
 }
 
+function hrefVariablesFromParameters(namespace, parameters) {
+  const path = parameters.get('path')
+    ? parameters.get('path')
+    : new namespace.elements.HrefVariables();
+
+  const query = parameters.get('query')
+    ? parameters.get('query')
+    : new namespace.elements.HrefVariables();
+
+  if (!path.isEmpty || !query.isEmpty) {
+    return path.concat(query);
+  }
+
+  return undefined;
+}
+
+function hrefFromParameters(path, queryParameters) {
+  const href = path.clone();
+  const queryString = queryParameters.keys().join(',');
```

Comment:
What if the parameter names need encoding before they become a part of the URI Template?

Reply:
E.g. `something[]`

Reply:
That's not yet supported, unsupported characters will emit an error in the parameter parser so they won't reach this part of the code yet.

Reply:
ok, didn't see FIXME, but maybe there's an issue already